client/components/disputed-order-notice/index.js

```diff
@@ -108,6 +108,90 @@ const DisputedOrderNoticeHandler = ( { chargeId, onDisableOrderRefund } ) => {
 	);
 };
 
+const UrgentDisputeNoticeBody = ( {
+	isPreDisputeInquiry,
+	disputeReason,
+	formattedAmount,
+	dueBy,
+	countdownDays,
+} ) => {
+	const formatString = isPreDisputeInquiry
+		? __(
+				// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+				"Please resolve the inquiry on this order for %1$s labeled '%2$s' by %3$s.",
```

Comment:
```suggestion
				"Please resolve the inquiry on this order of %1$s labeled '%2$s' by %3$s.",
```

Looking at the screenshot from https://github.com/Automattic/woocommerce-payments/issues/7568, it says 'of' instead of 'for'.
![Screenshot 2023-10-27 at 12 57 24 AM](https://github.com/Automattic/woocommerce-payments/assets/99638528/9b67fb89-66d9-4c56-a910-8fda166da0b5)

Just pointing out that inconsistency. I am not sure grammatically which one is the correct one.

Reply:
`of` reads better to my eyes, thanks 🙌